tools/pyorac/swath.py

```diff
@@ -450,26 +448,26 @@ def set_cldflag(self, size=3, low_res=False, cld_limit=3):
                              mode='same', boundary='symm') / num
             e_x2 = convolve2d(aot_tmp * aot_tmp, kernel,
                               mode='same', boundary='symm') / num
-            stdev = np.ma.masked_invalid(np.sqrt(e_x2 - e_x*e_x))
+            stdev = np.ma.masked_invalid(np.sqrt(e_x2 - e_x * e_x))
             self._cldflag[stdev > 0.1] += CLDFLAG["stdev"]
 
-        def opening_test(data, kernel, limit):
+        def opening_test(data, ikernel, limit):
```

Comment:
Since when is kernel a protected term?
And I use an i prefix to indicate indices/counters. Here I'd prefer to shorten the variable to `kern` if there's a clash

Reply:
Yeah, it's not. The problem that it was shadowing the same name in the outer scope. I then accidentally changed both to `ikernel`. I've now changed the function variable name to `kern` and left the outer scope variable as `kernel`.